src/pages/workspaces/workspace/Analyses.js

```diff
@@ -0,0 +1,469 @@
+import * as clipboard from 'clipboard-polyfill/text'
+import _ from 'lodash/fp'
+import * as qs from 'qs'
+import { Fragment, useEffect, useState } from 'react'
+import { a, div, h, img, label } from 'react-hyperscript-helpers'
+import * as breadcrumbs from 'src/components/breadcrumbs'
+import { requesterPaysWrapper, withRequesterPaysHandler } from 'src/components/bucket-utils'
+import { ViewToggleButtons, withViewToggle } from 'src/components/CardsListToggle'
+import {
+  ButtonPrimary,
+  Clickable,
+  IdContainer,
+  Link,
+  makeMenuIcon,
+  MenuButton,
+  PageBox,
+  Select,
+  spinnerOverlay
+} from 'src/components/common'
+import Dropzone from 'src/components/Dropzone'
+import { icon } from 'src/components/icons'
+import { DelayedSearchInput } from 'src/components/input'
+import { AnalysisDeleter, AnalysisDuplicator, findPotentialNotebookLockers, getTool, getDisplayName, getFileName, NotebookCreator, notebookLockHash } from 'src/components/notebook-utils'
+import PopupTrigger from 'src/components/PopupTrigger'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import galaxyLogo from 'src/images/galaxy-logo.png'
+import jupyterLogo from 'src/images/jupyter-logo.svg'
+import rLogo from 'src/images/r-logo.svg'
+import rstudioLogo from 'src/images/rstudio-logo.svg'
+import { Ajax } from 'src/libs/ajax'
+import colors from 'src/libs/colors'
+import { reportError, withErrorReporting } from 'src/libs/error'
+import * as Nav from 'src/libs/nav'
+import { notify } from 'src/libs/notifications'
+import { authStore } from 'src/libs/state'
+import * as StateHistory from 'src/libs/state-history'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import ExportAnalysisModal from 'src/pages/workspaces/workspace/notebooks/ExportNotebookModal'
+import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
+
+
+const analysisCardCommonStyles = listView => _.merge({ display: 'flex' },
+  listView ?
+    {
+      marginBottom: '0.5rem',
+      flexDirection: 'row',
+      alignItems: 'center'
+    } :
+    {
+      margin: '0 2.5rem 2.5rem 0',
+      height: 100,
+      width: 400,
+      flexDirection: 'column',
+      padding: 0
+    }
+)
+
+const noWrite = 'You do not have access to modify this workspace.'
+
+const sortTokens = {
+  lowerCaseName: notebook => notebook.name.toLowerCase()
```

Comment:
rename

Reply:
`Workflows.js` has an identical `sortTokens` functions. Does it make sense to move it to a common place so we have a single copy instead of three?